MAUI/PDF-Viewer/Toolbar.md

```diff
@@ -47,7 +47,25 @@ To customize the toolbar, you can programmatically add, remove, and hide the ite
 
 ### Hide specific toolbar
 
```

Comment:
Please ensure the documentation meets this triaging https://support.syncfusion.com/agent/tickets/610999#update-6362579